.github/workflows/mingw.yml

```diff
@@ -0,0 +1,42 @@
+name: MinGW
+on:
+  pull_request:
+    branches: [ '*' ]
+  push:
+    branches: [ '*' ]
```

Comment:
Why pushes? This also seems inconsistent in our other GitHub workflows.

Reply:
6 of the 8 workflows run on pushes. Running on "push" allows it to run after a PR has been merged to main with a rebase.